### ISSUE TITLE
TypeError: this.flash is not a function

### ISSUE THREAD

**Issue by AmirrezaNasiri** [keen-on-design/vue-flash-message#13]
Hi,
I try to use vue-flash-message in my Vue.js project but I face with `TypeError: this.flash is not a function` error.

I have registered plugin using:
_main.js:_
```
import Vue from 'vue'
import VueFlashMessage from 'vue-flash-message';
Vue.use(VueFlashMessage);
...
```

And used methods like:
_myComponent.vue:_
```
<template>
    <flash-message transitionName="flash" class="flashpool"></flash-message>
    <div @click="doSomething">hello</div>
</template>
<script>
    export default {
        name: 'something',
        data() {
            return {
                ...
            }
        },
        methods: {
            doSomething() {
                this.success('Your password has been updated.'); // TypeError: this.flash is not a function
                this.flash('Your password has been updated.', 'success'); // Again, TypeError: this.flash is not a function
            }
        }
    }
</script>
```

I also tried to rename default flash method and disabling shortcut creation; not working at all.
I think problem is with registering methods. How should I fix it?

> vue: 2.5.16
> vue-flash-message: 0.6.0

**Comment by keen-on-design** [comment]
HI @AmirrezaNasiri ! I have tested vue-flash-message with vue 2.5.6 but did not get any signs of an issue. I've patched v-f-m to 0.6.2 by the way).
The issue can be caused by the fact that your component is registered on a different Vue Instance (if you have more than one). Please also check if you have added a root element for flashes somwhere close to the root of your app layout. If none helps - please provide more code, or a link to your repo.

**Comment by dapobelieve** [comment]
Hello i have this issue too, how did you resolve it?
